code/game/objects/items/weapons/bee_briefcase.dm

```diff
@@ -0,0 +1,91 @@
+
+/obj/item/weapon/bee_briefcase
+	name = "briefcase"
+	desc = "This briefcase has easy-release clasps and smells vaguely of honey and blood..."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "briefcase"
+	item_state = "briefcase"
+	flags = CONDUCT
+	hitsound = "swing_hit"
+	force = 10
+	throw_speed = 2
+	throw_range = 4
+	w_class = 4
+	attack_verb = list("bashed", "battered", "bludgeoned", "thrashed", "whacked")
+	var/bees_left = 10
+	var/list/blood_list = list()
+	var/sound_file = 'sound/misc/briefcase_bees.ogg'
+	var/sound_cooldown = 0
+
+/obj/item/weapon/bee_briefcase/examine(mob/user)
+	..()
+	if(loc == user)
+		if(bees_left)
+			to_chat(user, "<span class='warning'>There are [bees_left] bees still inside in briefcase!</span>")
+		else
+			to_chat(user, "<span class='danger'>The bees are gone... Colony collapse disorder?</span>")
+
+/obj/item/weapon/bee_briefcase/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/reagent_containers/syringe))
+
+		var/obj/item/weapon/reagent_containers/syringe/S = I
+		if(!bees_left)
+			to_chat(user, "<span class='warning'>The briefcase is empty, so there is no point in injecting something into it.</span>")
+			return
+		if(S.reagents && S.reagents.total_volume)
+			to_chat(user, "<span class='notice'>You inject [src] with [S].</span>")
+			for(var/datum/reagent/A in S.reagents.reagent_list)
+				if(A.id == "blood")
+					if(!(A.data["donor"] in blood_list))
+						blood_list += A.data["donor"]
+				if(A.id == "strange_reagent")		//RELOAD THE BEES (1 bee per 1 unit, max 15 bees)
+					if(bees_left < 15)
+						bees_left = min(15, round((bees_left + A.volume), 1))	//No partial bees, max 15 bees in case at any given time
+						to_chat(user, "<span class='warning'>The buzzing inside the briefcase intensifies as new bees form inside.</span>")
+					else
+						to_chat(user, "<span class='warning'>The buzzing inside the briefcase swells momentarily, then returns to normal. Guess it was too cramped...</span>")
+				S.reagents.clear_reagents()
+				S.update_icon()
+
+	else if(istype(I, /obj/item/weapon/plantspray))
+		var/obj/item/weapon/plantspray/PS = I
+		user.drop_item(PS)
+		bees_left -= PS.pest_kill_str
+		if(bees_left < 0)
+			bees_left = 0
+		to_chat(user, "You spray [PS] into \the [src].")
+		playsound(loc, 'sound/effects/spray3.ogg', 50, 1, -6)
+		qdel(PS)
+
+/obj/item/weapon/bee_briefcase/attack_self(mob/user as mob)
+	if(!bees_left)
+		to_chat(user, "<span class='danger'>The lack of all and any bees at this event has been somewhat of a let-down...</span>")
+		return
+	else
+		if(!sound_cooldown)		//The cooldown doesn't prevent us from releasing bees, just stops the sound
+			sound_cooldown = 1
+			spawn(90)
+				sound_cooldown = 0
+			//Play sound through the station intercomms, so everyone knows the doom you have wrought.
+			var/list/intercomms = list()
+			for(var/obj/item/device/radio/intercom/I in world)
```

Comment:
Yea, i just copied code from the admin's play sound via intercomm verb for that bit. Definitely not the ideal way of doing it, but it works and these shouldn't see large enough use to really make a major impact on performance, though it still should ideally be cleaned up (same for the verb i copied from)